src/app_service/service/community_tokens/service.nim

```diff
@@ -364,6 +373,19 @@ QtObject:
     except RpcException:
         error "Error getting all community tokens", message = getCurrentExceptionMsg()
 
+  proc removeCommunityToken*(self: Service, communityId: string, chainId: int, address: string) =
+    try:
+      let response = tokens_backend.removeCommunityToken(chainId, address)
+      if response.error != nil:
+        let error = Json.decode($response.error, RpcError)
+        raise newException(RpcException, "error removing community token: " & error.message)
+        return
```

Comment:
```suggestion
```

Reply:
This will be addressed in https://github.com/status-im/status-desktop/pull/11668/commits/87f304a715a84b05c0b1dc78131e1d602467c1de which is already approved